caddyconfig/httpcaddyfile/options.go

```diff
@@ -43,6 +43,7 @@ func init() {
 	RegisterGlobalOption("local_certs", parseOptTrue)
 	RegisterGlobalOption("key_type", parseOptSingleString)
 	RegisterGlobalOption("auto_https", parseOptAutoHTTPS)
+	RegisterGlobalOption("servers", parseServerOptions)
```

Comment:
A reminder that we should discuss whether to rename this to "sockets" or "listeners".

Reply:
I don't like the alternatives as much. When talking about request matchers, we don't do like `match @foo`, we do `handle @foo`.

For the same reason, I don't think it makes sense to say `listeners [<listener_matcher>]`. What we actually configure is the server options (as in `http.Server`), and matching them by their listener values.